kinases-in-chembl/kinases_in_chembl.ipynb

```diff
@@ -24,7 +24,7 @@
   },
```

Comment:
<p>969 in the newest ChEMBL</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openkinome/kinodata/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>